docs/self-managed/components/console/configuration/configuration.md

```diff
@@ -56,6 +56,14 @@ These settings are useful when the application needs to make outgoing network re
 The proxy-related environment variables are lowercase because they follow a widely accepted convention used in many system environments and tools.
 :::
 
+### Experimental Features
+
+The following environment variables enable experimental features:
+
+| Environment variable                          | Description                                                                              | Example value | Default value |
+| --------------------------------------------- | ---------------------------------------------------------------------------------------- | ------------- | ------------- |
+| `CAMUNDA_CONSOLE_EXPERIMENTAL_DISCOVERY_MODE` | This mode allows orchestration clusters to register itself by calling an API in Console. | `true`        | `false`       |
```

Comment:
Could we link to the Orchestration cluster doc so it's clear how to configure it

Reply:
Suggest alternative text:
"This mode enables an experimental Console feature that integrates with the Orchestration cluster configuration <link>. When turned on, Orchestration clusters could report their basic configuration details to the Console, giving you a complete view of all Camunda deployments in your organization."